event-config.h.cmake

```diff
@@ -53,8 +53,8 @@
 /* Define to 1 if you have the `arc4random_buf' function. */
 #cmakedefine EVENT__HAVE_ARC4RANDOM_BUF 1
 
-/* Define to 1 if you have the `arc4random_addrandom' function. */
-#cmakedefine EVENT__HAVE_ARC4RANDOM_ADDRANDOM 1
+/* Define to 1 if you have the `arc4random_stir' function. */
+#cmakedefine EVENT__HAVE_ARC4RANDOM_STIR 1
```

Comment:
This will not work without changing cmake rules and don't forget about autotools

Reply:
I'm not understanding this. I've changed all the occurrences of EVENT__HAVE_ARC4RANDOM_ADDRANDOM in the repository.

Reply:
This is not how it works. You need to add the function `arc4random_stir` into https://github.com/libevent/libevent/blob/c6a74ee04fc5904fe443302d37798ac82a2016b3/CMakeLists.txt#L563 (actually replace this one with the new one)

Reply:
And same in `configure.ac`

Reply:
Ah I see. I've added the symbol in 54f621a6dbcd8921210bfe11cb7a7dcdcd85cd98.
I didn't remove the previous one, since there is still a path that uses it (`#if !defined(EVENT__HAVE_ARC4RANDOM)`).

I've also sorted the symbols alphabetically.

Reply:
> I've also sorted the symbols alphabetically.

Given this code is for an RNG, it'd be good if you didn't make random, unrelated style changes to the code at the same time. At the very least, any changes should be in separate commits (with proper & complete commit messages), so it's clear what is changing, and how.

Reply:
I've rebased the changes, so now the alphabetical sorting isn't part of this PR.